checkbox-provider-ce-oem/bin/cpufreq_governors.py

```diff
@@ -774,6 +814,15 @@ def main():
         info.print_policies_list()
         return 0
 
+    if args.probe_module:
+        expected_governor = ["conservative", "powersave", "ondemand",
```

Comment:
it would be better if we put this variable as a global variable, so we could use it in unit test